Gemfile.template

```diff
@@ -13,6 +13,7 @@ gem "ruby-maven-libs", "~> 3", ">= 3.9.6.1"
 gem "logstash-output-elasticsearch", ">= 11.14.0"
 gem "polyglot", require: false
 gem "treetop", require: false
+gem "faraday", "~> 2", :require => false # due elastic-transport depending faraday '~> 2'
```

Comment:
Not quite following... Is `elastic-transport` not specifying a dependency correctly? Not sure why bundler would not be providing >= 2 if its a dependency?

Reply:
It seems like if `elastic-transport` requires faraday ~> 2 that gem should be specifying that.

Reply:
After 🚶‍♂️, I was questioning myself why faraday-v1 was consumed in 9.0?! I found reason that I didn't remove `faraday (~> 1)` [here](https://github.com/elastic/logstash/blob/9.0/Gemfile.jruby-3.1.lock.release#L878) 🤦 .
So, this change is unnecessary now. I have checked in 8.x branches as well that all consuming faraday-v2.
I will close this.